types/entity.go

```diff
@@ -33,10 +34,12 @@ func (e Entity) MarshalJSON() ([]byte, error) {
 		UID        ImplicitlyMarshaledEntityUID   `json:"uid"`
 		Parents    []ImplicitlyMarshaledEntityUID `json:"parents"`
 		Attributes Record                         `json:"attrs"`
+		Tags       Record                         `json:"tags"`
```

Comment:
The [Rust implementation](https://github.com/cedar-policy/cedar/blob/dd6b9f0e280e4ed5eb1bea33891e5f254f7185b5/cedar-policy-core/src/ast/entity.rs#L369) actually elides the field if it's empty, (presumably for backwards compatibility?). To match the behavior, we could use the `omitempty` option here.

Reply:
That's great to know (and it fixes some incompatibilities even in our codebase 😄)

Reply:
Mmm, right. Thanks for the reminder.